kotsadm/pkg/kurl/kurl_nodes.go

```diff
@@ -0,0 +1,116 @@
+package kurl
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	kuberneteserrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	metrics "k8s.io/metrics/pkg/client/clientset/versioned"
+)
+
+type KurlNodes struct {
```

Comment:
wanna move all this to `pkg/kurl/types`?

Reply:
would that be a new package?

Reply:
Ya. I like the pattern and we seem to be using it